kokoro/linux/aarch64/python_crosscompile_aarch64.sh

```diff
@@ -1,21 +1,28 @@
 #!/bin/bash
 #
-# Builds protobuf C++ with aarch64 crosscompiler and runs a basic set of tests under an emulator.
-# NOTE: This script is expected to run under the dockcross/linux-arm64 docker image.
+# Builds protobuf python including the C++ extension with aarch64 crosscompiler.
+# The outputs of this script are laid out so that we can later test them under an aarch64 emulator.
+# NOTE: This script is expected to run under the dockcross/manylinux2014-aarch64 docker image.
 
 set -ex
 
 PYTHON="/opt/python/cp38-cp38/bin/python"
 
 ./autogen.sh
-CXXFLAGS="-fPIC -g -O2" ./configure
+CXXFLAGS="-fPIC -g -O2" ./configure --host=aarch64
 make -j8
 
+# create a simple shell wrapper that runs crosscompiled protoc under qemu
+echo '#!/bin/bash' >protoc_qemu_wrapper.sh
+echo 'exec qemu-aarch64 "../src/protoc" "$@"' >>protoc_qemu_wrapper.sh
+chmod ugo+x protoc_qemu_wrapper.sh
```

Comment:
pull request says java, but this was also part of your python PR...

Reply:
It's rebased now (this PR depended on some files introduced by they python PR).